src/slurm_plugin/instance_manager.py

```diff
@@ -660,36 +657,24 @@ def _add_instances_for_resume_file(
         self._clear_unused_launched_instances()
 
         self._scaling_for_jobs_single_node(
-            job_list=slurm_resume_data.jobs_single_node_no_oversubscribe,
+            job_list=slurm_resume_data.jobs_single_node_no_oversubscribe
```

Comment:
At this stage, can SlurmResumeData contain a property `jobs_single_node` that already has combines both "oversubscribe" and "no oversubscribe".

Reply:
absolutely, I think we can drop the distinction between "oversubscribe" and "no oversubscribe", now that we are able to manage both types. I'm considering this for next PR